code/game/machinery/doors/airlock.dm

```diff
@@ -94,7 +94,7 @@
 			else if(src.density)
 				visible_message("<span class='alium'>\The [user] begins forcing \the [src] open!</span>")
 				if(do_after(user, 5 SECONDS,src))
-					playsound(src, 'sound/machines/door/airlock_creaking.ogg', 100, 1, volume_channel = VOLUME_CHANNEL_DOORS)
+					playsound(src, 'sound/machines/door/airlock_creaking.ogg', 100, 1, volume_channel = VOLUME_CHANNEL_DOORS) //Normal airlock creak since we're not destroying the door

```

Comment:
prolly needs a chompstation comment, since new comment that doesn't exist in original file?

Reply:
I would probably just delete this comment honestly

Reply:
Ill nuke the comment then, I just put that there for understanding reasons.

Reply:
> Ill nuke the comment then, I just put that there for understanding reasons.

Yea, if this was a _ch file this comment would be totally good, and even encouraged, but the problem arises with the whole pulling from upstream thing